pkcs11/gkm/test-data-asn1.c

```diff
@@ -41,6 +41,10 @@ typedef struct _EggAsn1xDef ASN1_ARRAY_TYPE;
 typedef struct _EggAsn1xDef asn1_static_node;
 #include "test.asn.h"
 
+#define TEST_STRING "test data to write and read in the ASN1 structur"
```

Comment:
thanks. Fixed in 4af798a3. I will squash it to the place later so I will not confuse your ongoing review.